riff-raff/app/controllers/Application.scala

```diff
@@ -85,12 +89,12 @@ class Menu(config: Config) {
       "Documentation",
       Seq(
         SingleMenuItem(
-          "Deployment Types",
-          routes.Application.documentation("magenta-lib/types")
+          "Configuring a project",
```

Comment:
This is a pretty common use-case, so I think it's worth moving it out of `Everything else`